egs2/librimix/tse1/conf/tuning/train_enh_tse_td_speakerbeam_4gpu_max.yaml

```diff
@@ -1,9 +1,9 @@
 optim: adam
-max_epoch: 400
+max_epoch: 100
 batch_type: folded
 batch_size: 16
 iterator_type: chunk
-chunk_length: 24000
+chunk_length: 48000
```

Comment:
Do you think it may be useful to mention the sampling rate used for this parameter?

Reply:
Yes, this is true. I think I'd better mention the sample rate in the file name.